src/js/components/Select/SelectContainer.js

```diff
@@ -294,7 +244,13 @@ const SelectContainer = forwardRef(
         let nextActiveIndex = activeIndex + 1;
         while (
           nextActiveIndex < options.length &&
-          isDisabled(nextActiveIndex)
+          checkDisabled(
```

Comment:
Suggest "checkDisabled" -> "isDisabled", to align more with "isSelected"

Reply:
Done